.github/persist.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+mkdir -p ${PERSIST_DIR}/ccache-cachedir
```

Comment:
A late addition, wrap the existing body in:
```
if [[ -d "${PERSIST_DIR}/ccache-cachedir" ]] then
    echo "The Ccache directory exists at ${PERSIST_DIR}/ccache-cachedir"
else
   echo "Creating Ccache directory at ${PERSIST_DIR}/ccache-cachedir"
  ... old stuff ...
fi
```
That way we can check the logs to verify that the directory isn't removed by the runner